packages/flutter/lib/src/material/text_field.dart

```diff
@@ -1369,6 +1369,9 @@ class _TextFieldState extends State<TextField> with RestorationMixin implements
   }
 }
 
+TextStyle _m2CounterErrorStyle(BuildContext context) =>
+    Theme.of(context).textTheme.caption!.copyWith(color:Theme.of(context).errorColor);
```

Comment:
Nit: space after the ':'

```suggestion
    Theme.of(context).textTheme.caption!.copyWith(color: Theme.of(context).errorColor);
```